modules/ding_event/ding_event.strongarm.inc

```diff
@@ -160,7 +202,7 @@ function ding_event_strongarm() {
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
   $strongarm->name = 'language_content_type_ding_event';
-  $strongarm->value = '0';
+  $strongarm->value = '2';
```

Comment:
What does this do?

Reply:
This shows that Multilingual support is enabled for this content type.
0 - Disabled
1 - Enabled
2 - Enabled with translation